### DISCUSSION TITLE
Update path from 1.21.1 to 1.32.2-1?

### DISCUSSION THREAD

**Discussion by dekiesel** [Koenkk/zigbee2mqtt#18773] [General]
Hi,

I am running z2m via docker and would like to update from 1.21.1 to 1.32.2-1. 

Are there mandatory updates that I have to install oder can I just use the new image and start it up?

Thanks!

**Comment by Koenkk** [comment]
Nope, just update!

**Reply by dekiesel** [reply]
Great, thank you!
